docs/ref/class-based-views/generic-date-based.txt

```diff
@@ -254,17 +254,17 @@ views for displaying drilldown pages for date-based data.
 
     **Example myapp/urls.py**::
 
-        from django.conf.urls import url
+        from django.urls import path
 
         from myapp.views import ArticleMonthArchiveView
 
         urlpatterns = [
             # Example: /2012/aug/
-            url(r'^(?P<year>[0-9]{4})/(?P<month>[-\w]+)/$',
+            path('<int:year>/<int:month>/',
                 ArticleMonthArchiveView.as_view(),
                 name="archive_month"),
             # Example: /2012/08/
```

Comment:
It's a bad sad that reversing this will probably give `/2012/8/` now, but can't test it.